docs/advanced/flash.md

```diff
@@ -118,3 +122,13 @@ They test all flash API functionality. To run the tests, use these commands:
 
 - FlashIAP: `mbed test -m TARGET_NAME -n tests-mbed_drivers-flashiap`.
 - Flash HAL: `mbed test -m TARGET_NAME -n tests-mbed_hal-flash`.
+
+## Troubleshooting
```

Comment:
Would it make more sense to title this section "Notes When Troubleshooting" or something similar because it's a list of things to remember as opposed to a list of steps to take when troubleshooting?